src/ducks/balance/Balance.jsx

```diff
@@ -345,8 +345,16 @@ class Balance extends PureComponent {
     const accounts = this.getAllAccounts()
     const triggers = triggersCollection.data
 
+    const hasNoAccounts =
+      accounts.filter(
+        a =>
+          a._id !== 'health_reimbursements' &&
+          a._id !== 'professional_reimbursements' &&
+          a._id !== 'others_reimbursements'
+      ) === 0
```

Comment:
We do not have a `isVirtualAccount` helper function somewhere ?

Reply:
Not found, I created it.

Reply:
I've checked quickly by grepping on "virtual" and found that normally virtual accounts should have a "virtual" property set to true.